pilot/pkg/xds/filters/filters.go

```diff
@@ -164,24 +164,10 @@ var (
 
 	HTTPMx = buildHTTPMxFilter()
 
-	CaptureTLS = &listener.Filter{
-		Name: "capture_tls",
+	IstioNetworkAuthenticationFilter = &listener.Filter{
```

Comment:
This change is breaking TLS PASSTHROUGH with east-west gateway.

Gateway spec:
```
  spec:
    selector:
      istio: eastwestgateway
    servers:
    - hosts:
      - '*'
      port:
        name: tls-istiod
        number: 15012
        protocol: tls
      tls:
        mode: PASSTHROUGH
    - hosts:
      - '*'
      port:
        name: tls-istiodwebhook
        number: 15017
        protocol: tls
      tls:
        mode: PASSTHROUGH
```

Istiod logs:
```
2023-01-06T19:00:53.030559Z	warn	delta	ADS:LDS: ACK ERROR istio-eastwestgateway-c8bdbfc65-ss5hh.istio-system-6 Internal:Error adding/updating listener(s) 0.0.0.0_15012: Didn't find a registered implementation for name: 'istio_authn'
0.0.0.0_15017: Didn't find a registered implementation for name: 'istio_authn'
```

/cc @kyessenov @howardjohn

Reply:
I deployed the gateway using `gen-eastwest-gateway.sh`. Is that missing an update? It's referenced in the docs for multicluster.

Reply:
This change is a cherry-pick from the master branch. Is it broken there? Ambient and multi-cluster might not be interacting well at the moment.

Reply:
I think it might have to do with the version of Envoy used with the east-west gw when deployed via `gen-eastwest-gateway.sh`. I built the code from source - HEAD of experimental-ambient. Without this change, the TLS PASSTHROUGH works fine.

@kyessenov could you help fix this?

Reply:
AFAIK there's just one envoy version used for all proxies. Experimental ambient is pre-alpha so things may be broken occassionally.

Reply:
East-west gateway is not a part of Ambient. That should continue to work, but with this change TLS PASSTHROUGH is broken. @howardjohn any idea?

```
2023-01-06T19:00:53.030559Z	warn	delta	ADS:LDS: ACK ERROR istio-eastwestgateway-c8bdbfc65-ss5hh.istio-system-6 Internal:Error adding/updating listener(s) 0.0.0.0_15012: Didn't find a registered implementation for name: 'istio_authn'
0.0.0.0_15017: Didn't find a registered implementation for name: 'istio_authn'
```

Reply:
I think you are just using the wrong image for EW gateway and not an updated envoy probably

Reply:
I am using IstioOperator:
```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  name: eastwest
spec:
  revision: ""
  profile: empty
  components:
    ingressGateways:
      - name: istio-eastwestgateway
        label:
          istio: eastwestgateway
          app: istio-eastwestgateway
        enabled: true
        k8s:
          service:
            ports:
              - name: status-port
                port: 15021
                targetPort: 15021
              - name: tls
                port: 15443
                targetPort: 15443
              - name: tls-istiod
                port: 15012
                targetPort: 15012
              - name: tls-webhook
                port: 15017
                targetPort: 15017
  values:
    gateways:
      istio-ingressgateway:
        injectionTemplate: gateway
```